### PR TITLE
Reversed ordering of Māori English Translation Equivalents on Gloss l…

### PR DESCRIPTION
**Pull request by jonholdsworth**
…ist page and Gloss detail page.

### CODE REVIEW

File: signbank/dictionary/templates/dictionary/public_gloss_detail.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h2>
                 <header>
                     <h4>{% blocktrans %}Translation equivalents{% endblocktrans %}</h4>
                 </header>
-                {% for translation_language, translations in translation_languages_and_translations %}
+                {% for translation_language, translations in translation_languages_and_translations reversed %}
```
Comment (joshmcarthur) [inline signbank/dictionary/templates/dictionary/public_gloss_detail.html:51]: Can you add a comment above this line please to explain why we reverse the translation languages? I presume by default it's ordered alphabetically by language name?

File: signbank/dictionary/templates/dictionary/public_gloss_list.html [hunk 2]
```diff
@@ -181,7 +181,7 @@ <h5><strong>{% blocktrans %}Copyright and license:{% endblocktrans %}</strong></
             {% endif %}
             </div>
             <div class="panel-footer">
-            {% for glosstranslations in obj.glosstranslations_set.all %}
+            {% for glosstranslations in obj.glosstranslations_set.all reversed %}
```
Comment (joshmcarthur) [inline signbank/dictionary/templates/dictionary/public_gloss_list.html:184]: Same here

**Review by joshmcarthur — CHANGES_REQUESTED** [review]
I've put in a request to put a comment in these templates to explain _why_ we reverse for future devs wondering why we are doing this. It's important we do this in places where we are changing the existing signbank code in a way that the output would be different, to make it easier on ourselves in the future.

Please also make sure you take a couple of minutes to make sure you craft a good PR description for reviewers. It sets up context for us to know what ticket the change is for, as well as anything special you're looking for feedback on. For something like this, before and after screenshots can be useful to show the visual change being introduced here.

File: signbank/dictionary/templates/dictionary/public_gloss_list.html (hunk 2 above)
Comment (jonholdsworth) [reply signbank/dictionary/templates/dictionary/public_gloss_list.html:184]: I'm going to delete this pull request and re-submit it, as I think there is possibly a better approach at the database level.

**Comment by jonholdsworth** [comment]
I'm going to delete this pull request and re-submit it, as I think there is possibly a better approach at the database level.

**Pull request closed without merging**